### PR TITLE
Add diagram for CREATE CONNECTION

### PR DESCRIPTION
**Pull request by bernscode**
Add diagram for CREATE CONNECTION.

## Info
- **Description**: 
Add diagram for CREATE CONNECTION.

- **Preview**: 
https://pr-867.d2fbku9n2b6wde.amplifyapp.com/docs/upcoming/sql-create-connection/

- **Related code PR**: 
N/A

- **Related doc issue**: 
Resolves no doc issue.

- **Notes**: 
[ Any additional information? ]

## Before merging
  - [ ] (For version-specific PR) I have selected the corresponding software version in **Milestone** and linked the related doc issue to this PR in **Development**.
  - [ ] I have acquired the approval from the owner (and optionally the reviewers) of the code PR and at least one tech writer (`bernscode`, `CharlieSYH`, `emile-00`, & `hengm3467`). 
  - [ ] I have checked the doc site preview, and the updated parts look good. <details><summary>How?</summary>Scroll down and open this link: image
</details>

### CODE REVIEW

File: docs/sql/commands/sql-create-connection.md [hunk 1]
```diff
@@ -18,6 +18,33 @@ WITH (
...
 ```
 
+
+import rr from '@theme/RailroadDiagram'
+
+export const svg = rr.Diagram(
+    rr.Stack(
+        rr.Sequence(
+            rr.Terminal('CREATE CONNECTION'),
+            rr.Optional(rr.Terminal('IF NOT EXIST')),
```
Comment (CharlieSYH) [inline docs/sql/commands/sql-create-connection.md]: I believe it's 'IF NOT EXISTS'. Missing an "S" here.

Comment (CharlieSYH) [reply docs/sql/commands/sql-create-connection.md]: Same for the code syntax.

**Review by CharlieSYH — DISMISSED** [review]

File: docs/sql/commands/sql-create-connection.md (hunk 1 above)
Comment (bernscode) [reply docs/sql/commands/sql-create-connection.md]: @CharlieSYH Good catch! I've made the update on the lates commit. Thanks!

**Review by hengm3467 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**